source/features/show-commit-details.tsx

```diff
@@ -0,0 +1,24 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
+	// Ideally, we would run this immediately on page load, but there
+	// seems to be a race condition. Delaying by 100ms makes it work
+	// consistently.
+	setTimeout(() => {
```

Comment:
I think you should use `element-ready` instead.

Reply:
The button exists in the HTML sent by the server. I think there are other elements that need to be created before the click will work, but I haven't been able to determine what they are.